Dockerfile

```diff
@@ -44,6 +35,7 @@ RUN        npm ci --only=production
 # not trigger additional install as part of the build
 # in the following step.
 COPY       . /calypso/
+RUN        npm ci --only=production
 RUN        touch node_modules
```

Comment:
Do we still need this line? (If we don't, we should also update the above comment ^)